docs/ui.md

```diff
@@ -1,20 +1,40 @@
-# Kubernetes UI Instructions
+# Kubernetes User Interface
+Kubernetes has a web-based user interface that displays the current cluster state graphically. It is accessible via `https://<kubernetes-master>/ui`, which redirects to `https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kube-ui/#/dashboard/`.
```

Comment:
I would delete the second sentence, as it's redundant with the second sentence of the next section.